### ISSUE TITLE
about training expirence

### ISSUE THREAD

**Issue by windspire** [huangyangyu/SeqFace#5]
Thank you for your generous share , your model has very good robustness through large-scale data testing. I am trying to reiteration papers on resnet20 , but met some difficulties ,such as convergence difficulties， fine-tune failure. Could you share some training expirence for us , or  give some more detailed proposals  about how to prepare training data ,how to set params , perhaps there are some tricks ，easily neglected ，which make the training hard to convergence . 
Looking forward to your reply.

**Comment by huangyangyu** [comment]
Thanks for your attention. In large scale data training, you could use identity dataset to train a l2-sphereface to get a good base model, then using identity dataset and sequence dataset to train the seqface network. The identity dataset which we use is MS-Celeb-1M, we collect sequence dataset from all kinds of TV channels, each sequence contains 5-10 images, the noisy and duplicate images will be discarded.

**Comment by windspire** [comment]
thank you for prompt reply.
1.  how to define a base model to be good , what accuracy it should be achieve on training set.
2.  images in each sequence has the same label  , or  labeling  additive , in your paper 4.2 “faces in dataset B(5575 identities) is then randomly split into 32996", so different images for one people have different labels ,right?

**Comment by thuhuwei** [comment]
1. L2-sphereface can be trained as the base model to solve convergence problems in training, and it achieves about 99.5% accuracy on LFW in our experiments.
2. Right, only images in one sequence have the same label. We split the dataset just in order to get sequences from CASIA-Webface.

**Comment by jnulzl** [comment]
@huangyangyu 我按你的方法尝试训练，训练几万次后LSRO loss降到9.x，DSA loss降到0.x，不知道这样到底收敛没有？可否把你的训练log分享一下？
